src/backend/minio/minio_manager.py

```diff
@@ -14,10 +14,10 @@ class MinioManager:
     def __init__(self):
 
         try:
-            self.client=Minio(
-                os.getenv('MINIO_HOST','minio:9000'),
-                access_key=os.getenv('MINIO_ROOT_USER','minioadmin'),
-                secret_key=os.getenv('MINIO_ROOT_PASSWORD','minioadmin'),
+            self.client = Minio(
+                f"{os.getenv('MINIO_HOST', 'minio')}:9000",
```

Comment:
Make sure, you can change the below lines 

                f"{os.getenv('MINIO_HOST', 'minio')}:9000",

                
                like below
                
                minio: os.getenv('MINIO_SERVER_PORT')
                
                Do not use default values for the below lines
                
                                access_key=os.getenv('MINIO_ROOT_USER', 'minioadmin'),
                secret_key=os.getenv('MINIO_ROOT_PASSWORD', 'minioadmin'),